### ISSUE TITLE
Gerber's / CPL and BOM in standard format

### ISSUE THREAD

**Issue by tuctboh** [picosonic/bbc-fdc#37]
Hi,

Not an electronics/cad person, but wondered if there was any chance this would be put into a format JLC or PCBWay could just take and run with? 

Thanks, Tuc

**Comment by picosonic** [comment]
Yes the gerber files included are in a standard format that can be used by any PCB manufacturer.

**Comment by tuctboh** [comment]
@picosonic - Sorry, I was actually more talking about the BOM. So from what I see, there is a BOM.txt that just talks about the parts. I can't submit this to PCBWAY to get it assembled. Is there a way you can generate a standard compliant BOM file for assembly?
